app/pulp/app/models/task.py

```diff
@@ -98,31 +88,18 @@ class Task(Model):
 
     Fields:
 
-    :cvar group: The group this task belongs to
-    :type group: models.UUIDField
-
-    :cvar state: The state of the task
-    :type state: models.TextField
-
-    :cvar started_at: The time the task started executing
-    :type started_at: models.DateTimeField
-
-    :cvar finished_at: The time the task finished executing
-    :type finished_at: models.DateTimeField
-
-    :cvar non_fatal_errors: Dictionary of non-fatal errors that occurred while task was running.
-    :type non_fatal_errors: models.JSONField
-
-    :cvar result: Return value of the task
-    :type result: models.JSONField
+        group (models.UUIDField): The group this task belongs to
+        state (models.TextField): The state of the task
+        started_at (models.DateTimeField): The time the task started executing
+        finished_at (models.DateTimeField): The time the task finished executing
+        non_fatal_errors (pulp.app.fields.JSONField): Dictionary of non-fatal errors that occurred
```

Comment:
No, it could be that `pulp.app.fields isn't included in the apidoc generation.
(looking into it)